packages/gpx/gpx.1.0.1/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/rundat/ocaml-gpx/archive/v1.0.1.tar.gz"
```

Comment:
There's a dot missing here: the URL should be   
`https://github.com/rundat/ocaml-gpx/archive/v.1.0.1.tar.gz`
not
`https://github.com/rundat/ocaml-gpx/archive/v1.0.1.tar.gz`